numpy/core/fromnumeric.py

```diff
@@ -250,11 +250,9 @@ def reshape(a, newshape, order='C'):
      # Taking a view makes it possible to modify the shape without modifying
      # the initial object.
      >>> c = b.view()
```

Comment:
In light of @seberg's comment on the issue, what do you think about this, and then removing the `c.shape =` line?

```suggestion
     >>> c = b.reshape((20,))
```

Reply:
Yeah, but you would need to add a check that no copy was actually made if you want to advertise a way of reshaping while ensuring an error for a copy :/.

Reply:
At the top of the example it states " If you want an error to be raised when the data is copied, you should assign the new shape to the shape attribute of the array", but in the suggested commit we are not assigning a shape to that attribute directly anymore, should that statement also be changed then?

Reply:
If it is changed to the suggested commit, then the statement above the code example should be changed as well right?
_"If you want an error to be raised when the data is copied, you should assign the new shape to the shape attribute of the array"_
Because we are not directly assigning a new shape to the shape attribute of the array anymore, we are using reshape which will not cause an error if a copy is made.

Reply:
Should we perhaps just remove this entire example from the docs if it is not a good practice anyways?

Reply:
I would be fine with that.  I suspect this is useful to few power-users only anyway, and they can probably figure things out one way or another.

Reply:
Made a new commit to the same branch where the example is removed from the doc string (a bit new to PRs so feel free to correct any mistakes).

Reply:
+1 for removing the example. I'd leave the first sentence in the Notes section there though, as that's generally useful info.

Reply:
I agree and re-added the comment "It is not always possible to change the shape of an array without copying the data." to the third commit. If everyone agrees that this update is preferred, then feel free to merge ^^

I don't have access to my pc for a few days, so I won't be able to change this branch for a bit.